torch/csrc/jit/tensorexpr/mem_arena.cpp

```diff
@@ -5,51 +5,47 @@ namespace torch {
 namespace jit {
 namespace tensorexpr {
 
+thread_local KernelArena* current_arena = nullptr;
```

Comment:
Minor nit: I would recommend to hide this symbol, either through static, anonymous namespace, or a local function. Otherwise, functions in other translation unit can "extern" into it, without going through the accessor. It is important to make sure the only access is through the accessors.